README

```diff
@@ -0,0 +1,43 @@
+WELCOME TO THE XRAY-16 PROJECT!
+===============================
+This README file is for the xray-16 computer game engine.
+
+xray-16 is a development of the engine used in STALKER: Call of Pripyat game
```

Comment:
Maybe "xray-16 is an evolution" sounds better?

Reply:
Then "OpenXRay" is an evolution.. Since xray-16 is already an CoP's engine.

Reply:
I don't mind either way - it's just a 'courtesy' intro after all.

Development and evolution seem to me as pretty much synonymous in this case.

I was thinking of OpenXRay as more of a 'community' and of xray-16 as one of its projects - thus the phrasing.

Reply:
@Xottab-DUTY we need another version of xray-16. Maybe, OpenXRay 1.6 can be.. Aw, forget it, it's another question